Jenkinsfile

```diff
@@ -308,6 +309,56 @@ def buildDiffCoverage() {
     }
 }
 
+def buildPerformance() {
+  return {
+    // Select docker-cph-X.  We want docker, metal (brix) and only one executor
+    // (exclusive), if the machine changes also change REALM_BENCH_MACHID below
+    node('docker && brix && exclusive') {
+      getSourceArchive()
+
+      def gitTag = readGitTag()
+      def gitSha = readGitSha()
```

Comment:
It's only being used once in the whole Jenkinsfile and having it not defined helped figuring out this particular issue :)